src/shims/backtrace.rs

```diff
@@ -108,7 +108,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
 
         let lo = tcx.sess.source_map().lookup_char_pos(pos);
 
-        let filename = lo.file.name.to_string();
+        let filename = lo.file.name.prefer_local().to_string();
```

Comment:
So do backtraces printed by regular generated code is local or remapped paths? Since they are part of the binary I assume it's remapped paths; then for consistency shouldn't Miri also use remapped paths?

Reply:
The backtraces for `rustc` compiled program will display remapped paths. I just thought since miri doesn't produce binaries it's fine to display the actual path. But if we want to match `rustc`'s behaviour, I'll change it to remapped.

I guess (as of now) `miri` doesn't support path remapping? So either should produce the same local path.